libsolidity/interface/OptimiserSettings.h

```diff
@@ -59,6 +59,8 @@ struct OptimiserSettings
 		"]"
 		"jmul[jul] VcTOcul jmul";      // Make source short and pretty
 
+	static char constexpr DefaultYulOptimiserCleanupSteps[] = "fDnTOc";
```

Comment:
Since the above default optimization sequence is covered with appropriate comments, I'd like to comment this one as well. All I need is for someone to tell me what this step actually does :smile:

Reply:
Hm... yeah... what *does* it actually do :-D... ``BlockFlattener``, ``DeadCodeEliminator``, ``ControlFlowSimplifier``, ``LiteralRematerialiser``, ``ForLoopConditionOutOfBody``, ``CommonSubexpressionEliminator``...

It basically cleans up the mess the ``StackCompressor`` leaves behind :-).
We should maybe actually pretty much say that:

```
// Cleanup after stack compression.
```

Reply:
Unfortunately, that's only half-true, since only the "legacy code transform" pipeline actually runs the stack compressor before this (the newer "optimized code transform" gives better feedback to allow for a more precise, directed stack compression, which doesn't require any cleanup afterwards)...

Reply:
Ah, well, it has a descriptive name, let's just skip the comment.
It's basically part of the ``Make source short and pretty`` sequence - we just couldn't keep it in there, since the stack compressor would destroy it again.